Add CRI-O install vars to relnote

adellape opened the pull request:
Trello: https://trello.com/c/39LSatb0

@ashcrow @giuseppe PTAL at this CRI-O section:

http://file.rdu.redhat.com/~adellape/111617/crio37relnote/release_notes/ocp_3_7_release_notes.html#ocp-37-crio

(@ahardin-rh You don't have to merge this, I can close this and rebase later after https://github.com/openshift/openshift-docs/pull/6008 lands. Just using your branch for a nicer diff.)

ahardin-rh committed:
Added OCP 3.7 Release Notes

adellape committed:
Add CRI-O install vars to relnote

ashcrow commented:
Responded via IRC to your questions @adellape. Will review after meetings 😄.

ashcrow commented on release_notes/ocp_3_7_release_notes.adoc:
```diff
@@ -85,6 +85,28 @@ CRI-O brings:
+openshift_use_crio=True
+----
+
+This setting pulls the *openshift3/cri-o:latest* image from the link:https://access.redhat.com/containers[Red Hat Registry]. It also sets
```
It's not latest on the enterprise install. It will use the ``openshift_image_tag`` which is sensed in the installer or set in the inventory.

adellape replied:
OK, will just drop `:latest` here.

ashcrow commented on release_notes/ocp_3_7_release_notes.adoc:
```diff
@@ -85,6 +85,28 @@ CRI-O brings:
+----
+
+This setting pulls the *openshift3/cri-o:latest* image from the link:https://access.redhat.com/containers[Red Hat Registry]. It also sets
+`openshift_docker_systemcontainer_image_registry_override` as its override just
```
Did you mean ``openshift_crio_systemcontainer_image_override``?

adellape replied:
It mentions `openshift_docker_systemcontainer_image_registry_override` in https://github.com/openshift/openshift-ansible/blob/master/inventory/byo/hosts.example#L118.

adellape replied:
I had also mention to talk about `openshift_crio_systemcontainer_image_override`, so I've now replaced the stuff from https://github.com/openshift/openshift-ansible/blob/master/inventory/byo/hosts.example#L118 with talk about overriding the default image instead. If the line from https://github.com/openshift/openshift-ansible/blob/master/inventory/byo/hosts.example#L118 can be ignored, I'll just leave it out.

ashcrow commented on release_notes/ocp_3_7_release_notes.adoc:
```diff
@@ -85,6 +85,28 @@ CRI-O brings:
+
+[NOTE]
+====
+{product-title} node hosts must be RPM-based or system containers when using
```
This is probably fine, but it's really the ``atomic-enterprise-node``/``origin-node`` service that can not be a docker container (but can be rpm or system container installed).

giuseppe commented:
LGTM with the changes @ashcrow suggested

adellape commented:
@ashcrow @giuseppe Updated per feedback and also added some info on the optional garbage collection settings.

Updated preview: http://file.rdu.redhat.com/~adellape/111617/crio37relnote/release_notes/ocp_3_7_release_notes.html#ocp-37-crio

sdodson commented on release_notes/ocp_3_7_release_notes.adoc:
```diff
@@ -85,6 +85,56 @@ CRI-O brings:
+
+[NOTE]
+====
+The `atomic-enterprise-node` service must be RPM- or system container-based when
```
atomic-openshift-node

ashcrow commented:
Looks good. I'm not sure if this is the place that needs the Tech Preview label but it should be noted somewhere.

adellape commented:
@ashcrow It is noted at the beginning of this CRI-O section (before my new additions in this PR), and also in the Tech Preview list later in the topic:

http://file.rdu.redhat.com/~adellape/111617/crio37relnote/release_notes/ocp_3_7_release_notes.html#ocp-37-technology-preview

ganhuang commented on release_notes/ocp_3_7_release_notes.adoc:
```diff
@@ -85,6 +85,56 @@ CRI-O brings:
+following:
+
+----
+openshift_crio_docker_gc_node_selector: {'runtime': 'cri-o'}
```
Ansible inventory doesn't support such grammar. Should be corrected to:

```
openshift_crio_docker_gc_node_selector={'runtime': 'cri-o'}
```

adellape commented:
Closing in favor of https://github.com/openshift/openshift-docs/pull/6525.
